locomotor/__init__.py

```diff
@@ -476,6 +480,8 @@ def process_Call(self, node, code, indent, loops):
             elif node.func.id == 'len':
                 assert len(node.args) == 1
                 line = '#' + args
+            elif node.func.id == 'print':
+                line = 'print(%s)' % args
```

Comment:
`print` can't really be handled this way since it should be printed at the client and not at the server. So you'd really have to record what the server says should be printed and then print it on the client later.